File: app/src/main/java/com/example/ibrahim/servisinfo_ib150071/data/KompanijePregledVM.java
package com.example.ibrahim.servisinfo_ib150071.data;

import java.io.Serializable;
import java.util.List;

/**
 * Created by Ibrahim on 25.7.2018..
 */

public class KompanijePregledVM implements Serializable {
    public static class Row implements Serializable
    {
        public int KompanijaID ;
        public String Naziv ;
        public String Adresa;
        public String Telefon ;
        public String Email ;
        public String KorisickoIme;
        public String LozinkaSalt ;
        public String LozinkaHash ;
        public int GradID ;
        public String Grad ;

    }

    public List<Row> rows;
}

File: RKS_IB150071_WebServisi/RKS_IB150071_WebServisi/Model2/KompanijePregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RKS_IB150071_WebServisi.Model2
{
    public class KompanijePregledVM
    {
        public class Row
        {
            public int KompanijaID { get; set; }
            public string Naziv { get; set; }
            public string Adresa { get; set; }
            public string Telefon { get; set; }
            public string Email { get; set; }
            public string KorisickoIme { get; set; }
            public string LozinkaSalt { get; set; }
            public string LozinkaHash { get; set; }
            public int GradID { get; set; }
            public string Grad { get; set; }

        }
        public List<Row> rows;

    }
}